pkg/actions/lua/lakefs/client.go

```diff
@@ -183,6 +183,46 @@ func OpenClient(l *lua.State, ctx context.Context, user *model.User, server *htt
 				l.PushString(rr.Body.String())
 				return 2
 			}},
+			{Name: "stat_object", Function: func(state *lua.State) int {
+				repo := lua.CheckString(l, 1)
+				ref := lua.CheckString(l, 2)
+				reqURL, err := url.JoinPath("/repositories", repo, "refs", ref, "objects", "stat")
+				if err != nil {
+					check(l, err)
+				}
+				req, err := newLakeFSJSONRequest(ctx, user, http.MethodGet, reqURL, nil)
+				if err != nil {
+					check(l, err)
+				}
+				// query params
+				q := req.URL.Query()
+				q.Add("path", lua.CheckString(l, 3))
+				req.URL.RawQuery = q.Encode()
+				rr := httptest.NewRecorder()
+				server.Handler.ServeHTTP(rr, req)
+				l.PushInteger(rr.Code)
+				l.PushString(rr.Body.String())
+				return 2
+			}},
+			{Name: "get_repo", Function: func(state *lua.State) int {
+				repo := lua.CheckString(l, 1)
+				reqURL, err := url.JoinPath("/repositories", repo)
+				if err != nil {
+					check(l, err)
+				}
+				req, err := newLakeFSJSONRequest(ctx, user, http.MethodGet, reqURL, nil)
+				if err != nil {
+					check(l, err)
+				}
+				// query params
+				q := req.URL.Query()
+				req.URL.RawQuery = q.Encode()
+				rr := httptest.NewRecorder()
+				server.Handler.ServeHTTP(rr, req)
+				l.PushInteger(rr.Code)
+				l.PushString(rr.Body.String())
+				return 2
```

Comment:
not sure we need that function (see comment about getting storage_namespace from action global) in usage of this function in lua.